app.css

```diff
@@ -1,79 +1,73 @@
 .eager-self-summary {
   display: block;
-  -webkit-font-smoothing: antialiased; /* TODO Talk about */
-  padding: 2em;
-  margin: 1em 0;
+  padding: 1em;
 }
 
-eager-wrapper {
+.eager-self-summary eager-wrapper {
   display: block;
-  max-width: 500px;
 }
 
-eager-wrapper[data-position="left"]{
+@media (min-width: 500px) {
+  .eager-self-summary eager-wrapper {
+    max-width: 450px;
+  }
+}
+
+.eager-self-summary[data-position="left"] eager-wrapper {
   margin-right: auto;
 }
 
-eager-wrapper[data-position="center"]{
+.eager-self-summary[data-position="center"] eager-wrapper {
```

Comment:
Nesting the element selectors under `.eager-self-summary` prevents other eager apps from having conflicting styles.